src/components/Button.js

```diff
@@ -1,6 +1,24 @@
-import React from 'react';
-import '../components/Buttons.css';
+import styled from '@emotion/styled';
 
-export default function Button(props) {
-  return <button className={props.className || 'button'} {...props} />;
-}
+const Button = styled.button`
+  background-color: rgba(164, 173, 235, 1);
+  border-radius: 35px;
+  width: ${props => (props.btnType === 'nav' ? '85%' : '60%')};
```

Comment:
try not to use abbreviations like `btn`. I know it's all over the internets, but it could lead to stuff like `e` and `i` ...